File: EXAMEN/TABLAMULTIPLICAR.f
      PROGRAM TABLAMULTIPLICAR
C     TABLA DE MULTIPLICAR EN FORTRAN 77 (FORMATO FIJO .F)
      INTEGER N, I
      CHARACTER*1 PAUSA

      WRITE(*,*) 'Introduce un numero:'
      READ(*,*) N

      WRITE(*,*) 'Tabla de multiplicar del ', N

      DO 100 I = 1, 10
         WRITE(*,'(I3, A, I2, A, I5)') N, ' x ', I, ' = ', N*I
100   CONTINUE

      WRITE(*,*) 'Presiona ENTER para salir...'
      READ(*,'(A)') PAUSA
      END

File: EXAMEN/TablaMultiplicar.pas
program TablaMultiplicar;
uses crt;

var
  n, i: integer;

begin
  clrscr;
  write('Introduce un numero: ');
  readln(n);

  writeln('Tabla de multiplicar del ', n);
  for i := 1 to 10 do
    writeln(n, ' x ', i, ' = ', n * i);

  writeln('Presiona ENTER para salir...');
  readln;
end.
